src/vs/workbench/api/common/extHost.api.impl.ts

```diff
@@ -985,7 +985,7 @@ export function createApiFactoryAndRegisterActors(accessor: ServicesAccessor): I
 
 				return extHostWorkspace.findTextInFiles(query, options || {}, callback, extension.identifier, token);
 			},
-			findTextInFilesNew: (query: vscode.TextSearchQueryNew, options?: vscode.FindTextInFilesOptionsNew, token?: vscode.CancellationToken): vscode.FindTextInFilesResponse => {
+			findTextInFilesNew: (query: vscode.TextSearchQuery, options?: vscode.FindTextInFilesOptions, token?: vscode.CancellationToken): vscode.FindTextInFilesResponse => {
```

Comment:
I kept the `New` here because it has a different return type in the new call, so I can't overload it.

Reply:
Instead of using `New`, I think we typically use a version number. So `findTextInFilesNew2`, `findTextInFilesNew3`, ...

Reply:
ok - so then should the `Old` ones be called `Old`?

Reply:
Also, should I rename the filenames to have `2` instead of `New`?